client/src/components/edit/index.js

```diff
@@ -1,35 +1,224 @@
 import React, { Component } from 'react'
-import Experience from './Experience'
-import './edit.css'
-import { Personal } from './Personal'
-import { Media } from './Media'
-import { style } from '../../widgets/styles'
+import PropTypes from 'prop-types'
 import Avatar from 'material-ui/Avatar'
-import Person from 'material-ui/svg-icons/social/person'
+import { RaisedButton } from 'material-ui'
+
+import Personal from '../../containers/personal'
+import Media from '../../containers/media'
+import { style } from '../../widgets/styles'
+import Experiences from './Experiences'
+
+import './edit.css'
+
+function clickFunction(submit) {
+  submit('personal')
+  submit('media')
+  submit('existingExperiences')
+  submit('newExperiences') // post
+}
 
 class EditProfile extends Component {
   render() {
+    const {
+      profileImage,
+      submit,
+      createThings,
+      updateUserInfo,
+      updateUserExperience,
+      updateUserVideo,
+      videoId,
+      userId,
+      loading
+    } = this.props
+
+    const submitHandler = values => {
+      if (!values.newExp) return
+
+
+      values.newExp.forEach(exp => {
+        const {
+          name,
+          position,
+          current,
+          award,
+          startDate,
+          endDate,
+          address,
+          programs
+        } = exp
+
+        let addressLine_1, addressLine_2, city, state, postalCode, country;
```

Comment:
This section needs to be changed a few ways. Let me know when a good time is so we can work through it:
- Variables and object keys should be camelCase. This was added in the newest commit of setting up google places autocomplete.
- item types should be different. We can look at the Google Places address component types together to figure them out.
- Address line 1 shouldn't be using the formatted address.

Reply:
the '_' in addressLine_1 and 2 are required to pass the info to the backend. There isn't a way to transform addressLine1 -> address_line_1 using the camelize keys framework